runtime/src/snapshot_utils.rs

```diff
@@ -53,18 +55,42 @@ pub struct SnapshotArchiveInfo {
     /// Archive format for the snapshot file
     pub archive_format: ArchiveFormat,
 }
+
+/// Information about an incremental snapshot archive: its path, slot, base slot, hash, and archive format
+pub struct IncrementalSnapshotArchiveInfo {
+    /// Path to the incremental snapshot archive file
+    pub path: PathBuf,
+
+    /// The slot that the incremental snapshot was based from.  This is the same as the full
+    /// snapshot slot used when making the incremental snapshot.
+    pub base_slot: Slot,
+
+    /// Slot that the incremental snapshot was made
+    pub slot: Slot,
+
+    /// Hash of the accounts at this slot
+    pub hash: Hash,
+
+    /// Archive format for the incremental snapshot file
+    pub archive_format: ArchiveFormat,
```

Comment:
does it make sense to just have `IncrementalSnapshotArchiveInfo` hold a `SnapshotArchiveInfo` and a `base_slot`

Reply:
I second that -- it does look a lot of fields have the same names and have same meanings.

Reply:
Love it. Will do!

Reply:
Done.